_data/other.yml

```diff
@@ -285,6 +285,12 @@ websites:
       software: Yes
       doc: http://doxfer.webmin.com/Webmin/Enhanced_Authentication
 
+    - name: Wikia
+      url: https://www.wikia.com/
```

Comment:
I wasn't sure which to put here - `www.wikia.com/fandom` is their kinda new project to relate more to the people making and maintaining the wikis whereas `http://community.wikia.com` is their "starting point" into making wikis, so to speak.